app/helpers/application_helper.rb

```diff
@@ -50,5 +50,13 @@ def avatar_uri(member, size = 150)
       default: :identicon
     })
   end
+
+  # Returns a string with the quantity and the right pluralization for a
+  # given collection and model.
+  def localize_plural(collection, model)
+    size       = collection.size
+    model_name = model.model_name.human(:count => size)
```

Comment:
We swapped over to ruby 2.0 syntax recent, so don't be afraid to use it :) Rubocop offers ways to automatically fix this for you too